src/i18n/de.js

```diff
@@ -0,0 +1,43 @@
+export default {
+    header: {
+        slogan: 'Die Zukunft der Web-Entwicklung.'
+    },
+    callToAction: {
+        start: 'Jetzt loslegen',
+        join: 'Am Chat teilnehmen'
+    },
+    nav: {
+        code: 'Code',
+        cloud: 'Cloud',
+        docs: 'Doku'
+    },
+    bio: {
+        first: 'Asynchrone, Ereignis-gesteuerte Architektur die auf Apples <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO</a> basiert und <a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">höchste Performance</a> liefert.',
+        second: 'In <a target="_blank" href="https://swift.org">Swift</a> geschrieben, der Sprache, die nicht nur performant, sondern auch einfach zu erlernen ist.',
+        third: 'Ausdrucksstarkes, <a target="_blank" href="https://developer.apple.com/videos/play/wwdc2015/408/">Protokollorientiertes Design</a> mit Fokus auf Typsicherheit und einfach zu wartendem Code.'
+    },
+    contribute: {
+        title: 'Mitmachen!',
+        options: {
+            code: {
+                description: 'Schlage Features vor, melde Bugs, und integriere deinen Code auf GitHub!'
+            },
+            chat: {
+                title: 'Am Chat teilnehmen',
+                description: 'Hilf Neulingen beim Einstieg, und starte mit Vapor.'
+            },
+            donate: {
+                title: 'Werde Unterstützer',
```

Comment:
I'd suggest to keep it in the same style as `Am Chat teilnehmen`, so: `Unterstützer werden`.

Reply:
Good suggestion! Fixed.